src/components/Card.vue

```diff
@@ -1,138 +1,148 @@
 <template>
-<div class="home-cards-item">
-  <div class="home-card">
+  <div class="home-cards-item">
+    <div class="home-card">
       <header class="home-card-topbar"></header>
       <div class="home-card-image">
-          <img :src="quest.image" alt="quest_image">
+        <img :src="quest.image" alt="quest_image" />
       </div>
       <div class="home-card-content">
-          <div class="home-card-title">{{quest.name}}</div>
-          <table class="home-card-text">
-              <tr>
-                  <td>
-                      <img class="home-card-icon" src="../assets/images/vectors/level.png"><span>{{quest.level}}</span>
-                 </td>
-
-                  <td>
-                      <img class="home-card-icon" src="../assets/images/vectors/hours.png"><span>{{quest.hours}}</span><span> hours</span>
-                 </td>
-             </tr>
-              <tr>
-                  <td>
-                      <img class="home-card-icon" src="../assets/images/vectors/credits.png"><span>{{quest.credits}}</span><span> credits</span>
-                  </td>
-                  <td>
-                      <img class="home-card-icon" src="../assets/images/vectors/steps.png"><span>{{quest.steps}}</span><span> steps</span>
-                  </td>
-              </tr>
-            </table>
+        <div class="home-card-title">{{ quest.name }}</div>
+        <table class="home-card-text">
+          <tr>
+            <td>
+              <img
+                class="home-card-icon"
+                src="../assets/images/vectors/level.png"
+              /><span>{{ quest.level }}</span>
+            </td>
+
+            <td>
+              <img
+                class="home-card-icon"
+                src="../assets/images/vectors/hours.png"
+              /><span>{{ quest.hours }}</span
+              ><span> hours</span>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <img
+                class="home-card-icon"
+                src="../assets/images/vectors/credits.png"
+              /><span>{{ quest.credits }}</span
+              ><span> credits</span>
+            </td>
+            <td>
+              <img
+                class="home-card-icon"
+                src="../assets/images/vectors/steps.png"
+              /><span>{{ quest.steps }}</span
+              ><span> steps</span>
+            </td>
+          </tr>
+        </table>
         <div class="home-card-btn-holder">
-            <form target="_blank" action="https://www.qwiklabs.com/quests/34">
```

Comment:
the past developers implemented it with a form tag lets minimize the code using an anchor tag instead